sklearn/plot/tests/test_heatmap.py

```diff
@@ -0,0 +1,24 @@
+from sklearn.plot import plot_heatmap
+from sklearn.utils.testing import SkipTest
+import numpy as np
+
+
+def test_heatmap():
+    try:
+        import matplotlib
+    except ImportError:
+        raise SkipTest("Not testing plot_heatmap, matplotlib not installed.")
+
+    import matplotlib.pyplot as plt
```

Comment:
yea this seems to be a problem because my tests cover much more than what Codecov is showing. i wasn't able to check coverage locally as I'm using a mac and running matplotlib from terminal is giving errors which I'm not able to resolve yet.